### DISCUSSION TITLE
Clarification regarding phoneme training

### DISCUSSION THREAD

**Discussion by BoneGoat** [coqui-ai/TTS#850] [General Q&A] [answered]
Hi, I'm training a Swedish model and I can't figure out if I'm doing this wrong or not. In the docs it says: "If you use phonemes for training and your language is supported here, you don’t need to set your character list."

So I don't set a character list in my config as sv-se is a supported language. However, the generated config.json in the output folder still adds the characters section with the default characters. Is this being ignored or do I still need to override this using find_unique_chars.py?

`
"use_phonemes": true,
    "use_espeak_phonemes": true,
    "phoneme_language": "sv-se",
    "compute_input_seq_cache": true,
    "text_cleaner": "swedish_cleaners",
    "enable_eos_bos_chars": false,
    "test_sentences_file": "",
    "phoneme_cache_path": "/root/data/eva-tts/phoneme_cache",
    "characters": {
        "pad": "_",
        "eos": "~",
        "bos": "^",
        "characters": "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz!'(),-.:;? ",
        "punctuations": "!'(),-.:;? ",
        "phonemes": "iy\u0268\u0289\u026fu\u026a\u028f\u028ae\u00f8\u0258\u0259\u0275\u0264o\u025b\u0153\u025c\u025e\u028c\u0254\u00e6\u0250a\u0276\u0251\u0252\u1d7b\u0298\u0253\u01c0\u0257\u01c3\u0284\u01c2\u0260\u01c1\u029bpbtd\u0288\u0256c\u025fk\u0261q\u0262\u0294\u0274\u014b\u0272\u0273n\u0271m\u0299r\u0280\u2c71\u027e\u027d\u0278\u03b2fv\u03b8\u00f0sz\u0283\u0292\u0282\u0290\u00e7\u029dx\u0263\u03c7\u0281\u0127\u0295h\u0266\u026c\u026e\u028b\u0279\u027bj\u0270l\u026d\u028e\u029f\u02c8\u02cc\u02d0\u02d1\u028dw\u0265\u029c\u02a2\u02a1\u0255\u0291\u027a\u0267\u02b2\u025a\u02de\u026b"
    }
`

Best regards

**Comment by erogol** [comment, marked as answer]
These are the characters by default used by the model. It needs to be saved for using the model later. So everything is fine.

**Comment by Ca-ressemble-a-du-fake** [comment]
Hi @BoneGoat , 
It's been a year since you posted this question but maybe you remember :wink: ! Did you actually set the `characters` and `punctuation` lists to `None` ?
